app/models/scored_schools_list.rb

```diff
@@ -17,6 +17,10 @@ def position(school)
     schools_with_positions.select {|_position, schools| schools.include?(school)}.first.first
   end
 
+  def top_three
+    with_points.schools_at(0, 3)
```

Comment:
for readability (and if it's not used in too many places) maybe the `schools_at` method could have a little refactor to use keyword arguments e.g.
```
  def schools_at(index:, length:)
    @scored_schools.slice(index, length)
  end
```
so here it would read
```
  def top_three
    with_points.schools_at(index: 0, length: 3)
  end
```